test/test_notebook_document.py

```diff
@@ -582,3 +582,76 @@ def test_notebook__did_close(
         )
         wait_for_condition(lambda: mock_notify.call_count >= 2)
         assert len(server.workspace.documents) == 0
+
+
+def test_notebook_definition(
+    client_server_pair,
+):  # pylint: disable=redefined-outer-name
```

Comment:
```suggestion
def test_notebook_definition(client_server_pair):
```

This shouldn't be necessary after PR #418.

Reply:
@ccordoba12 - done!